activetable/static/js/src/activetable.js

```diff
@@ -138,16 +138,22 @@ function ActiveTableXBlock(runtime, element, init_args) {
     }
 
     function createCell(oldCell, index){
-        var cellId = oldCell.id
-        var type = $(oldCell).find("input").attr("type");
-        var height = $(oldCell).find("input").outerHeight();
+        var cellId = oldCell.id;
+        var kind = "textarea";
+
+        if ($(oldCell).find("input").length) {
+            var kind = "input";
+        }
+
+        var type = $(oldCell).find(kind).attr("type");
+        var height = $("#input_" + cellId).outerHeight();
         var newId = cellId.replace("_"+index+"_", "_"+(index+1)+"_")
         var cell = $("<td>").attr("id", newId);
 
         if (!cellId.endsWith("_0")){
             cell.addClass("active unchecked");
-            var cellInput = $("<input>").attr("id", "input_"+newId).attr("type", type).css("height", height+"px");
-            if (!type.localeCompare("radio")) {
+            var cellInput = $("<" + kind + ">").attr("id", "input_"+newId).attr("type", type).css("height", height+"px");
+            if (type != null && !type.localeCompare("radio")) {
```

Comment:
What do you expect from 'type'?, could it be if (type && !type.localeCompare("radio"))...? @andrey-canon